constraint/pkg/client/client.go

```diff
@@ -596,6 +591,15 @@ func (c *Client) Dump(ctx context.Context) (string, error) {
 	return c.driver.Dump(ctx)
 }
 
+func (c *Client) GetDescriptionForStat(statName string) string {
```

Comment:
add in `source` as first arg and use it for the engine type casting; no switch; there will be a map from driver name --> driver obj; consider making a keyable `source` struct, something that works as a map key

Reply:
I will take another pass at using the `d.Name` from https://github.com/open-policy-agent/frameworks/pull/293 once that goes in but leaving as is for now.